UPGRADING.md

```diff
@@ -0,0 +1,7 @@
+UPGRADING
+=========
+
+Upgrade from version 2.1.\* to 2.2.\*
```

Comment:
Because Markdown interprets asterisks as text in italics, so I needed to escape them